### PR TITLE
Adds cookie compliance banner

### PR DESCRIPTION
**Pull request by rasummer**
Gives user the ability to opt out of specific groups of cookies and prevents the loading of JS libraries and iFramed content based on these choices.

@TimMoore Ready for your review.

### CODE REVIEW

**Comment by ignasi35** [comment]
Hi @rasummer this looks good. I'm testing it locally on a Mac OSX/Chrome and I get the alert on every page I visit (even after allowing cookies) though. I tried accessing the page using `localhost` and `example.com`(which I mapped to `127.0.0.1`. I would expect the cookie disclaimer to disappear for a month (or forever) once I allowed.

File: src/main/twirl/com/lightbend/lagom/docs/main.scala.html [hunk 1]
```diff
@@ -15,6 +15,22 @@
         <meta name="viewport" content="width=device-width,maximum-scale=1"/>
         <link href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" rel="stylesheet" type="text/css"/>
         <link rel="stylesheet" href="@context.path/@context.assetFingerPrint-all-styles-concat.css" type="text/css">
+        <!-- OneTrust Cookies Consent Notice (Production Standard, lagomframework.com, en-GB) start -->
+        <script src="https://optanon.blob.core.windows.net/consent/1ee3f0d6-9ed5-4b9e-bed7-ce6faaeaca5d.js" type="text/javascript" charset="UTF-8"></script>
```
Comment (rasummer) [inline src/main/twirl/com/lightbend/lagom/docs/main.scala.html]: @ignasi35 When testing locally you'll need to swap this include out for the testing version: `<script src="https://optanon.blob.core.windows.net/consent/1ee3f0d6-9ed5-4b9e-bed7-ce6faaeaca5d-test.js" type="text/javascript" charset="UTF-8"></script>`

**Comment by rasummer** [comment]
@ignasi35 To make sure local testing works fine you'll need to use the following script for the main.scala.html file, line 19 : `<script src="https://optanon.blob.core.windows.net/consent/1ee3f0d6-9ed5-4b9e-bed7-ce6faaeaca5d-test.js" type="text/javascript" charset="UTF-8"></script>`

**Comment by rasummer** [comment]
@ignasi35 I will run this locally again to confirm and see if I can reproduce. You're correct in that the banner should go away once you Allow Cookies. Please note that this solution itself uses cookies that store your preferences, these are cookies that fall under the Strictly Necessary category. If you are using your browser to completely block all cookies, then you  will see the banner each time.

**Comment by ignasi35** [comment]
Hey @rasummer I tested again with your suggestions and works as expected. I noticed the PR is now blocked by a conflict. If we get that sorted out I'm happy to merge.

**Comment by ignasi35** [comment]
Use `template` please.

This repository uses GitHub pages and we reserve `master` for the materialization of all the contents of the website.

**Comment by rasummer** [comment]
Got it. 

For some reason when I pull in the upstream version of template and try to merge into my branch, I don't get a conflict locally. Not sure why, but in this case could you be kind enough to resolve the conflict for me by fixing the index.scala.html page in the GitHub UI to use the below code (lines 41 & 42), and remove the old iframe code on lines 44 and 45. 

`<div class="yt-widget flex-video widescreen" id="yt-d1jT0UOVx9U">
                    @cookieVideoWarning()`

**Comment by rasummer** [comment]
Actually ignore my last comment, GitHub UI let me make adjustments, should be good now.

File: src/main/twirl/com/lightbend/lagom/docs/index.scala.html [hunk 2]
```diff
@@ -37,8 +37,8 @@ <h1>Meet Lagom</h1>
 
                 <p>Watch the 3-minute <i>Meet Lagom</i> video below, or try Lagom yourself. <a class="" href="@context.path/get-started.html" target="_blank">Create a Lagom project and test a simple Hello World service</a> in just a few minutes.</p>
 
-                <div id="meet-lagom" class="flex-video widescreen">
-                    <iframe id="player" class="youtube-embed-player" src="//www.youtube.com/embed/d1jT0UOVx9U?rel=0&amp;autohide=1&amp;modestbranding=0&amp;showinfo=1" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+                <div class="yt-widget flex-video widescreen" id="yt-d1jT0UOVx9U">
+                    @cookieVideoWarning()
```
Comment (ignasi35) [inline src/main/twirl/com/lightbend/lagom/docs/index.scala.html]: This is black magic.

**Comment by ignasi35** [comment]
thanks @rasummer

**Comment by rasummer** [comment]
Thanks ;)

**Pull request merged**